tests/pthread/test_pthread_locale.c

```diff
@@ -52,10 +52,7 @@ int main (int argc, char *argv[])
     pthread_join(thread, (void**)&child_loc);
   }
 
-#ifdef REPORT_RESULT
-  int result = (main_loc == child_loc);
-  REPORT_RESULT(result);
-#endif
 
-  pthread_exit(NULL);
```

Comment:
We don't need this anymore?

Reply:
Calling pthread_exit from `main` here is just odd..  it might work but it looks pretty out of place.   I don't see why it was ever here.